password/password.cabal

```diff
@@ -46,6 +47,7 @@ library
     , cryptonite  >= 0.15.1   && < 0.27
     , memory      >= 0.14     && < 0.16
     , text        >= 1.2.2    && < 1.3
+    , tasty-quickcheck
```

Comment:
Leaning on `tasty-quickcheck` in the library is unnecessary. The `Arbitrary` instances can be put in the tests (best to put them in a separate module that has `{-# OPTIONS_GHC -Wno-orphans #-}` in it)